src/lib/math.js

```diff
@@ -314,6 +314,47 @@ const $builtinmodule = function (name) {
         throw new Sk.builtin.NotImplementedError("math.isqrt() is not yet implemented in Skulpt");
     };
 
+    function lcm(...args) {
+        function abs(n) {
+            return JSBI.lessThan(result, JSBI.__ZERO)
+                ? new Sk.builtin.int_(JSBI.unaryMinus(result))
+                : new Sk.builtin.int_(result);
+        }
```

Comment:
We need to apply this differently if `n` is a `number`.

```suggestion
        function abs(n) {
            if (typeof n === "number") {
                return Math.abs(n);
            }
            return JSBI.lessThan(n, JSBI.__ZERO) ? new Sk.builtin.int_(JSBI.unaryMinus(n)) : new Sk.builtin.int_(n);
        }
```